python/tests/requirements.txt

```diff
@@ -1,3 +1,3 @@
 OpenNMT-py==1.1.*
-OpenNMT-tf==2.9.*
+OpenNMT-tf==2.11.*
```

Comment:
I suppose this is for Python 3.8 compatibility. Is it possible to just skip testing some Python versions?

Reply:
It seems cibuildwheel doesn't support that: https://cibuildwheel.readthedocs.io/en/stable/options/
I'll create a shell script that calls pytest and does nothing if Python >= 3.8.